tailwind.all.css

```diff
@@ -20,4 +21,3 @@
 @import "resources/css/srs_cards.css";
 @import "resources/css/tabler-extension.css";
 @import-glob "src/main/frontend/**/[!_]*.css";
-@import "tailwindcss/utilities";
```

Comment:
@xyhp915 Shouldn't tailwind utilities be loaded before our main css, so we can easily override things?